examples/custom-server/src/app/_components/RichText/Serialize/index.tsx

```diff
@@ -0,0 +1,169 @@
+            text = <em key={i}>{text}</em>
+          }
+
+          if (node.underline) {
+            text = (
+              <span style={{ textDecoration: 'underline' }} key={i}>
+                {text}
+              </span>
+            )
+          }
+
+          if (node.strikethrough) {
+            text = (
+              <span style={{ textDecoration: 'line-through' }} key={i}>
+                {text}
+              </span>
+            )
+          }
+
+          return <Fragment key={i}>{text}</Fragment>
+        }
+
+        if (!node) {
+          return null
+        }
+
+        if (
+          customRenderers &&
+          customRenderers[node.type] &&
+          typeof customRenderers[node.type] === 'function'
+        ) {
+          return customRenderers[node.type]({ node, Serialize, index: i })
+        }
+
+        switch (node.type) {
+          case 'br':
+            return <br key={i} />
+          case 'h1':
+            return (
+              <h1 key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </h1>
+            )
+          case 'h2':
+            return (
+              <h2 key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </h2>
+            )
+          case 'h3':
+            return (
+              <h3 key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </h3>
+            )
+          case 'h4':
+            return (
+              <h4 key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </h4>
+            )
+          case 'h5':
+            return (
+              <h5 key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </h5>
+            )
+          case 'h6':
+            return (
+              <h6 key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </h6>
+            )
+          case 'quote':
+            return (
+              <blockquote key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </blockquote>
+            )
+          case 'ul':
+            return (
+              <ul key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </ul>
+            )
+          case 'ol':
+            return (
+              <ol key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </ol>
+            )
+          case 'li':
+            return (
+              <li key={i}>
+                <Serialize content={node.children} customRenderers={customRenderers} />
+              </li>
+            )
+
+          case 'link':
+            return (
+              <a
```

Comment:
It should use the <Link> tag

Reply:
Which tag? `next/link`? The repo tries to be as simple as possible in order to _only_ demonstrate the core feature so some of this is intentionally simple.

Reply:
Done!